### DISCUSSION TITLE
Modelleerprincipes - P2. Objecttypes zijn herbruikbaar

### DISCUSSION THREAD

**Discussion by pmaria** [Geonovum/disgeo-imsor#59] [General] [answered]
Modelleerprincipe [[P2]](https://geonovum.github.io/disgeo-imsor/modelleerprincipes/#p2-objecttypes-zijn-herbruikbaar) -Objecttypes zijn (her)bruikbaar in verschillende registraties.

Discussie uit expertgroepsessie:

> - [Voorstel voor aanpassing van principe]: om het concreter te maken kunnen we zeggen 'objecttypes moeten dezelfde naam hebben, of naar dezelfde concepten verwijzen.'
> - Paul: lastig, bij mij hangen kenmerken samen met de objecttypes, het lijkt alsof dat hier ook weer los wordt getrokken. 
> - Er zijn twee verschillende views: je zou kunnen zeggen dat je dingen kan combineren als ze hetzelfde objecttype hebben (via een 'sameAs' relatie, bv), of dat dingen combineerbaar zijn als hun objecttypes gebaseerd zijn op hetzelfde concept.
> - Michel: Het zou verhelderend zijn als we een voorbeeld (bv van de BAG en BGT) kunnen maken met de twee mogelijke links of afgestemde IDs/afgestemde klassen en de rol van IMSOR daarin.
> - Lennart: Lijkt me handig als we de verschillende soorten samenhang kunnen presenteren met voorbeelden (semantische samenhang, logische samenhang, etc). 
> - Twee 'paden' voor samenhang worden onderscheiden: een pad waarbij semantische verbanden en algoritmes/specificaties een grote rol spelen, en een pad waarbij URIs/IDs die instanties linken de voorkeur heeft.

**Comment by lvdbrink** [comment]
Dit zou een mooi onderwerp zijn om eens in een experimentele setting mee aan de slag te gaan.

**Comment by pmaria** [comment, marked as answer]
Relateerbaar toegevoegd aan stelling. Dit principe is afgeleid van P1 waarover in #57 ook discussie is gevoerd.

**Comment by lvdbrink** [comment]
P2 is vastgesteld in [expertgroep 8 juli](https://github.com/Geonovum/disgeo-imsor/blob/master/overleg/08-07-2021-aantekeningen.md).
